### ISSUE TITLE
response time calculation

### ISSUE THREAD

**Issue by kwevej** [pglass/how-do-i-locust#3]
The text:
_Locust will then compute and store the duration from the time the request was made until the time the .success() function was called._

Is not right. Only the original POST / GET response time is taken into account
If you need to have the real response time reported, you need to do modify it before reporting as:
```
 resp.locust_request_meta["response_time"] += timediff
 resp.success()
```
where timediff is 
```
 timediff = 1000 * (time.monotonic() - result_startt)

**Comment by pglass** [comment]
Ah, thanks. You're right. Looking back through my old code, I used a separate "success" function to calculate the total elapsed time since the start time.

**Comment by pglass** [comment]
@kwevej Addressed in #4
